source/index.html

```diff
@@ -1,12 +1,240 @@
+              </svg>
+              <span class="visually-hidden">Вконтакте</span>
+            </a>
+          </li>
+          <li class="footer-social__item">
+            <a class="button-social__link button-social__link--youtube"
+              href="https://www.youtube.com/user/htmlacademyru">
+              <svg class="social__icon" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg"
+                width="24" height="19" fill="none" viewBox="0 0 24 19">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                  d="m15.85 8.8-5.24-3.5a.65.65 0 0 0-1.01.55v6.98a.65.65 0 0 0 1.01.54l5.24-3.49a.65.65 0 0 0 0-1.09Zm-4.94 2.8V7.08l3.4 2.27-3.4 2.27Zm12.48-8.56a2.4 2.4 0 0 0-1.45-1.63C18.22-.02 12.25 0 12 0S5.78-.02 2.06 1.4A2.4 2.4 0 0 0 .61 3.05 25.32 25.32 0 0 0 0 9.34c0 3.26.33 5.22.61 6.3a2.4 2.4 0 0 0 1.45 1.63c3.72 1.43 9.69 1.4 9.94 1.4h.08c.75 0 6.31-.04 9.86-1.4a2.4 2.4 0 0 0 1.45-1.64c.28-1.07.61-3.03.61-6.3 0-3.25-.33-5.22-.61-6.29Zm-1.27 12.27a1.1 1.1 0 0 1-.65.74c-3.5 1.34-9.4 1.32-9.47 1.32-.07 0-5.98.02-9.47-1.32a1.09 1.09 0 0 1-.65-.74c-.27-1-.57-2.85-.57-5.97 0-3.12.3-4.97.57-5.97a1.1 1.1 0 0 1 .65-.74C6.03 1.29 11.93 1.3 12 1.3c.07 0 5.98-.02 9.47 1.32a1.1 1.1 0 0 1 .65.74c.27 1 .57 2.85.57 5.97 0 3.12-.3 4.96-.57 5.97Z" />
+              </svg>
+              <span class="visually-hidden">Ютуб</span>
+            </a>
+          </li>
+          <li class="footer-social__item">
+            <a class="button-social__link button-social__link--telegram" href="https://t.me/htmlacademy">
+              <svg class="social__icon" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg"
+                width="20" height="18" fill="none" viewBox="0 0 20 18">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                  d="M19.46.15c.18.15.27.38.23.61l-2.63 15.76a.66.66 0 0 1-1.01.44l-7.88-5.25a.66.66 0 0 1-.15-.96l2.61-3.26-4.36 2.9a.66.66 0 0 1-.65.05L.36 7.8a.66.66 0 0 1 .08-1.2L18.82.03a.66.66 0 0 1 .64.11ZM2.33 7.32 5.86 9.1l7.56-5.04a.66.66 0 0 1 .88.96l-4.8 6 6.43 4.3 2.28-13.66L2.33 7.32Z" />
+              </svg>
+              <span class="visually-hidden">Телеграм</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="developer">
+      <div class="developer_wrapper">
+        <h3 class="visually-hidden">Сайт разработчика</h3>
+        <div class="htmlacademy__wrapper">
```

Comment:
```suggestion
        <div class="developer__wrapper">
```
Дальше тоже поправить.